ui/main/src/app/modules/share/multi-select/multi-select.component.ts

```diff
@@ -31,7 +31,10 @@ export class MultiSelectComponent implements AfterViewInit, OnDestroy, OnChanges
     @Output() selectionChange: EventEmitter<string[]> = new EventEmitter<string[]>();
 
     private oldOptions: Array<MultiSelectOption>;
-    private oldSelectedOptions: any;
+    // It is important to initialize it with an empty array an not undefined to avoid
+    // to trigger the change event when no option is selected on the first load
+    // See Issue #7526
+    private oldSelectedOptions: any = new Array<string>();
```

Comment:
There is a little typo in the comment, I suggest:
```
    // It is important to initialize it with an empty array and not undefined to avoid
    // triggering the change event when no option is selected on the first load
    // See Issue #7526
```
And I don't think it's necessary to keep the ": any" on the declaration of oldSelectedOptions